### DISCUSSION TITLE
The output only resembles 80% of the original.

### DISCUSSION THREAD

**Discussion by Thanh1122kkll** [glucauze/sd-webui-faceswaplab#134] [General]
I have tried various methods, including selecting the full option of the advanced swap mode (codeformer, ldsr, color correction, etc.), and even excluding the blend-face option. However, the output face still has some differences compared to the original. It only approximates about 80% of the original. I wonder if there is a way to achieve a 100% identical result to the original, just swapping then blending the face layer with new skin( like a filter). without adding any additional interpretation. I used a very simple prompt combined with 3 control nets, including lineart, to maximize the level of detail. thanks all

**Comment by LucianoDaluz** [comment]
How many photos are you using to create the checkpoint? Gotta tell you did using like 15 photos and the result are amazing.

**Reply by Thanh1122kkll** [reply]
hi, thanks for your help, can I ask you something, for better results,  should I create 15 copies of the original image or how should I do it? Because I don't have much data about that person?

**Comment by glucauze** [comment]
See my anwser here : https://github.com/glucauze/sd-webui-faceswaplab/discussions/135
